### PR TITLE
providers/aws: pass pointers directly to d.Set in RDS

### PR DESCRIPTION
**Pull request by phinze**
d.Set has a pointer nil check we can lean on

/cc @fanhaf

### CODE REVIEW

**Comment by fanhaf** [comment]
Thanks a lot!

:)
1 kwi 2015 16:23 "Paul Hinze" <notifications@github.com> napisał(a):

File: builtin/providers/aws/resource_aws_db_instance.go [hunk 1]
```diff
@@ -304,33 +304,38 @@ func resourceAwsDbInstanceRead(d *schema.ResourceData, meta interface{}) error {
...
+	}
+
+	if v.Endpoint != nil {
+		d.Set("port", v.Endpoint.Port)
+		d.Set("address", v.Endpoint.Address)
+
+		if v.Endpoint.Address != nil && v.Endpoint.Port != nil {
+			d.Set("endpoint",
+				fmt.Sprintf("%s:%d", *v.Endpoint.Address, *v.Endpoint.Port))
+		}
```
Comment (catsby) [inline builtin/providers/aws/resource_aws_db_instance.go]: Good catch(es)

**Comment by catsby** [comment]
LGTM

**Comment by mitchellh** [comment]
LGTM

**Pull request merged**